### DISCUSSION TITLE
Was Nuke Support extenssion removed from VS Code?

### DISCUSSION THREAD

**Discussion by BusHero** [nuke-build/nuke#1228] [Q&A] [answered]
I'm unable to find the Nuke Support extension in Visual Studio Code. Was it removed?
![image](https://github.com/nuke-build/nuke/assets/24370515/8d997bc4-9d92-4cba-ab91-25194a42f4cf)

**Comment by matkoch** [comment, marked as answer]
https://github.com/nuke-build/nuke/issues/1262
